Simply disabling the code which enables floating point exceptions
is probably wrong, but I don't have a replacement for the
non-posix functions fegetexcept(3) and feenableexcept(3).

--- src/Main/bootstrap.cxx	2015-02-18 21:10:05.000000000 +0100
+++ src/Main/bootstrap.cxx	2015-11-01 01:25:56.665212195 +0100
@@ -96,8 +96,12 @@
 initFPE (bool fpeAbort)
 {
+#if defined(__GLIBC__)
     if (fpeAbort) {
         int except = fegetexcept();
         feenableexcept(except | FE_DIVBYZERO | FE_INVALID);
     } else {
         signal(SIGFPE, handleFPE);
     }
+#else
+    signal(SIGFPE, handleFPE);
+#endif
